ArjunaJTA/jdbc/pom.xml

```diff
@@ -16,6 +16,9 @@
   <name>Narayana: ArjunaJTA jdbc</name>
   <description>transactional driver</description>
 
+  <properties>
+    <checkstyle.skip>true</checkstyle.skip>
```

Comment:
Does this need `<sortpom.skip>true</sortpom.skip>` adding too? If you look at the diff for this pull request and where `<checkstyle.skip>true</checkstyle.skip>` I think it's possible (unless it is inherited somehow for the various modules) that those modules which we have checkstyle.skip being true should also have `<sortpom.skip>true</sortpom.skip>` too. Do you agree that seems a correct default assumption?

Reply:
Not sure @tomjenkinson , strangly enough the sortpom was already enabled by default: https://github.com/jbosstm/narayana/blob/main/pom.xml#L177-L178
So I think modules that were not skipping before it shouldn't skip it now.

Reply:
At least now checkstyle and sortpom will be consistent, meaning that they will be both enabled by default.